src/app/(post-login)/submission/program/[shortName]/clinical-submission/page.tsx

```diff
@@ -65,6 +67,35 @@ import {
 } from './types';
 
 const ClinicalSubmission = ({ shortName }: { shortName: string }) => {
+	const {
+		data,
+		loading,
+		refetch,
+		updateQuery: updateClinicalSubmissionQuery,
+	} = useQuery(CLINICAL_SUBMISSION_QUERY, {
+		variables: {
+			shortName,
+		},
+	});
+
+	const dataHandlers = {
+		refetch,
+		updateClinicalSubmissionQuery,
+	};
+
+	return loading ? (
+		<Loader />
+	) : (
+		<ClinicalSubmissionSection {...{ shortName, data, dataHandlers }} />
+	);
+};
```

Comment:
clean up. moved data fetching to a parent component. easier seperation of when we have data and when we don't - rather than checking 8 million times in 6 different places.